python/main.py

```diff
@@ -51,32 +76,100 @@ async def lifespan(app: FastAPI):
     allow_headers=["*"],
 )
 
-
-class HelloResponse(BaseModel):
+class AddItemResponse(BaseModel):
     message: str
 
-
-@app.get("/", response_model=HelloResponse)
-def hello():
-    return HelloResponse(**{"message": "Hello, world!"})
+class Item(BaseModel):
+    name: str
+    category: str
+    image_name: str
 
 
-class AddItemResponse(BaseModel):
-    message: str
 
+def insert_item(item: Item):
+    with sqlite3.connect(db) as conn:
```

Comment:
We don't need to connect to database every time we are inserting a value. That is unoptimised and can lead to memory issues. Instead, we should maintain a persistent connection to the database and use that